### DISCUSSION TITLE
PK SIM-10 Plotting

### DISCUSSION THREAD

**Discussion by alifele** [Open-Systems-Pharmacology/Forum#1048] [Questions & Problems]
Hello,

I have been watching a youtube tutorial about PK-SIM (they were using version 7).
in the final plot, their plot of concentration starts with an initial value equal to zero (following plot):
![image](https://user-images.githubusercontent.com/48031441/144938254-c65e4ae3-54c4-48ac-8d35-c0bba0f561ea.png)

But in my case, although it starts with from initial value equal to zero, my plot does not start from zero:
![image](https://user-images.githubusercontent.com/48031441/144938356-ee9d911b-1fd7-427a-aa43-9062eb0a0473.png)

You might suspect that this is simply because I have not set the initial value to be zero, but when I zoom in, the plot is not starting from zero:
![image](https://user-images.githubusercontent.com/48031441/144938467-002a4286-31c7-4a6b-b924-a64d7f21db4f.png)

i.e. it should be something like:
![image](https://user-images.githubusercontent.com/48031441/144938553-b913e521-87ee-4ddf-8c6c-33d644eba912.png)

But I don't know why my version of PK-SIM (version10) is not plotting the blue part!

Is this a software bug or do I just need to change some settings?

Thanks,

**Comment by msevestre** [comment]
It looks like you are plotting using a log scale. Try using a linear scale

**Reply by AndreDlm** [reply]
As @msevestre pointed out, a value of 0 cannot be plotted on a log scale (log 0 is undefined). If you increase the output resolution under settings between 0 and 0.05 h you might be able to see the increasing part of the curve:
![image](https://user-images.githubusercontent.com/26541603/145002787-57a7952d-5b30-4498-ba6c-09db731271ef.png)

**Comment by StephanSchaller** [comment]
@alifele , are you sure you set an *ORAL* administration? Looks more like an IV bolus.
